app/styles/discord.css

```diff
@@ -6,6 +6,19 @@
   height: 100%;
 }
 
+.linking__logo {
+  margin-top: 20px;
+  display: flex;
+}
+
+.linking__logo>.line {
+  width: 300px;
+  border-top: none;
+  position: relative;
+  bottom: 50%;
```

Comment:
This will also be removed then when doing align-items: center;

Reply:
since it is a div thus border bottom needs to be up so align items center wont work

Reply:
It will work try adding.
![image](https://github.com/Real-Dev-Squad/website-my/assets/85296770/e6de6556-5caa-4b04-98de-e481e1c92645)
![image](https://github.com/Real-Dev-Squad/website-my/assets/85296770/7cdab0f6-2618-4339-a902-70b61053ca66)